### ISSUE TITLE
[23:00:18] (ATTENTION) No usable PokeStops found in your area. Is your maximum distance too small?

### ISSUE THREAD

**Issue by rsngrd** [NECROBOTIO/NecroBot#5361]
Is this an IP ban? I can still log in on my phone so i know its not a total account ban. Anyone have information for me on how to set up a VPN

**Comment by x3r4n0** [comment]
its a IP Ban u have to wait about 1 hour to get in the game with the banned  IP.

Dont use more than 1 account per IP!

**Comment by rsngrd** [comment]
Hey thanks. So in about an hour, I should be able to reopen necrobot?

**Comment by rubadubdub9** [comment]
yeah it seems the moment i set up a second account (on a different computer but same ip address), i got ip banned on both accounts.

**Comment by rsngrd** [comment]
I dont get that, because I only ever set up 1 account
I can log in from my phone but its only when i turned on Necro Bot did i get the ban

**Comment by x3r4n0** [comment]
oh thats new to me i never heard about but maybe your config is weak, what about your speed u are moving?

**Comment by zzxxzzxxSG** [comment]
Cos you have Phone and PC under SAME IP.
So it's 2 accounts (even same account) on 1 IP.

**Comment by rsngrd** [comment]
I have my setting on very slow, only 7km per hour. Also I have enable
humanized throw on, I have sniping turned off and I set up my lat and long
to where I normally go out and play.

**Comment by x3r4n0** [comment]
turn off the wifi on your phone will solve your ban (not your ip ban from the wifi/WAN ip)

**Comment by rsngrd** [comment]
I see what you are saying, how can I avoid this, not use wifi network on my
phone that I'm using for necro? when logging into Pgo?

**Comment by x3r4n0** [comment]
@rsngrd use a vpn service (hide.me free trial) test if it works with necro if not its account softban not ipban

**Comment by x3r4n0** [comment]
@rsngrd i repeat u need different WAN-IPS (not Local IPs like 192.168.0.101) to use more than 1 accounts on same time. i hope is clearly

**Comment by rsngrd** [comment]
OK. I will try the hide me

**Comment by Ardevik** [comment]
I Get this if i use necrobot 0.8.5 or 0.8.6, if I use 0.8.1 everything is fine, this not ip ban or account ban? but the necrobot itself?

**Comment by rsngrd** [comment]
@x3r4n0 Hey there, so i tried to download hide.me. I got it set up and connected with a new IP. Now when I open NEcro, it is saying that I have Google 2 step verification enabled and won't work, I checked my google account settings and I don't have 2 step enabled

**Comment by rsngrd** [comment]
I'm pretty sure this is ip ban

**Comment by vijai1996** [comment]
I created a new PTC account and experiencing this right from start. Also, I set up sock5 proxy but the bot wont recognize it. I could use proxy on browser fine

**Comment by Linewalker** [comment]
https://github.com/NECROBOTIO/NecroBot/issues/5522#issuecomment-239429286
Please close. No code issue.
